src/content/docs/new-relic-solutions/get-started/networks.mdx

```diff
@@ -244,6 +244,27 @@ For more detail on specific agents and integrations, and about ports, keep readi
     </tr>
 </tbody>
 
+  <thead>
+   <tr>
+      <th>
+       **Lookups**
```

Comment:
Should this be `**NRQL Lookup Tables**` instead?

Reply:
I went with just "Lookup tables", we don't appear to preface with "NRQL" elsewhere.